### ISSUE TITLE
Problem in diversity of resulting style when testing

### ISSUE THREAD

**Issue by TimBdV** [NVlabs/SPADE#81]
While training a model on my own dataset the resulting generated images were close to the input data, but when testing the diversity of results drastically dropped with only one kind of style in the outputted image while the training dataset possessed at least 10.

I must say that I trained using the vae and for about 30 epochs with pretty good results in the training phase but not when using test.py (also using the vae). I used the default training  options.

I thought that the GAN was experiencing mode collapse so I tried lowering the learning rate, as GANs are really sensitive to it. Sadly the same problem occurred as the same style of images as before were outputted during the testing phase.

Did this happen to anybody else or have you an idea on the source of the problem?

**Comment by qiuyuchen14** [comment]
I'm facing the same issue here! I trained using vae on kitchen scene, and run test.py with vae multiple times, but the texture of the objects always looks the same: for example, the cabinet is always brown, I was expecting white or blue, which are shown in the training set.

**Comment by qiuyuchen14** [comment]
I think the problem might be the training dataset is small.  I tried to retrain the model on a bigger dataset, and now I can see different results.

**Comment by TimBdV** [comment]
> I commented "model.eval()" in test.py and retrained the model on a bigger dataset, and now I can see different results.

I also commented "model.eval()" but didn't retrain and finally I got some diversity in the results!
